mindmeld/converter/dialogflow.py

```diff
@@ -0,0 +1,375 @@
+                          "from mindmeld import Application",
+                          "app = Application(__name__)",
+                          "__all__ = ['app']"]
+
+            for info, spacing in zip(begin_info, [1, 2, 1, 1, 0]):
+                target.write(info + "\n" * spacing)
+
+            intents = self._get_file_names("intents")
+
+            # for i, (main, languages) in enumerate(intents.items()):
+            for i, main in enumerate(intents.keys()):
+
+                df_main = os.path.join(self.dialogflow_project_directory,
+                                       "intents", main + ".json")
+
+                with open(df_main) as source:
+                    if "usersays" in df_main:
+                        logger.error("Please check if your intent file"
+                                     "names are correctly labeled.")
+
+                    datastore = json.load(source)
+                    replies = []
+
+                    for response in datastore["responses"]:
+                        for message in response["messages"]:
+                            language = message["lang"]
+
+                            if 'speech' in message:
+                                data = message["speech"]
+
+                                replies = data if isinstance(data, list) else [data]
+
+                                if datastore["fallbackIntent"]:
+                                    function_name = "default" + "_" + language
+                                    if language == "en":
+                                        # TODO: support multiple defaults for languages
+                                        handles = ["default=True", "intent='unsupported'"]
+                                    else:
+                                        handles = ["intent='unsupported'"]
+                                else:
+                                    function_name = "renameMe" + str(i) + "_" + language
+                                    handles = ["intent=" + "'" +
+                                               self.clean_name(datastore["name"]) +
+                                               "_usersays_" + language + "'"]
+
+                                target.write("\n\n\n" +
+                                             self.create_function(handles=handles,
+                                                                  function_name=function_name,
+                                                                  replies=replies))
+            target.write("\n")
+
+    # =========================
+    # convert project
+    # =========================
+
+    def convert_project(self):
+        """ Converts a DialogFlow project into a MindMeld project.
+
+        DialogFlow projects consist of entities and intents.
+            note on languages:
+                DialogFlow supports multiple languages and locales. They store their training
```

Comment:
Refer to my earlier comment on languages, I would recommend removing it for now.